inspirehep/modules/workflows/views.py

```diff
@@ -376,3 +401,180 @@ def robotupload_callback():
+
+                $ curl \\
+                    http://web:5000/callback/workflows/resolve_validation_errors \\
+                    -H "Host: localhost:5000" \\
+                    -H "Content-Type: application/json" \\
+                    -d '{
+                        "_extra_data": {
+                            ... extra data content
+                        },
+                        "id": 910648,
+                        "metadata": {
+                            "$schema": "https://labs.inspirehep.net/schemas/records/hep.json",
+                            ... record content
+                        }
+                    }'
+
+            The response:
+
+                HTTP 200 OK
+
+                {"mesage": "Workflow 910648 validated, continuing it."}
+
+
+            A failed example:
+
+                $ curl \\
+                    http://web:5000/callback/workflows/resolve_validation_errors \\
+                    -H "Host: localhost:5000" \\
+                    -H "Content-Type: application/json" \\
+                    -d '{
+                        "_extra_data": {
+                            ... extra data content
+                        },
+                        "id": 910648,
+                        "metadata": {
+                            "$schema": "https://labs.inspirehep.net/schemas/records/hep.json",
+                            ... record content
+                        }
+                    }'
+
+            The error response will contain the workflow that was passed, with the
+            new validation errors:
+
+                HTTP 400 Bad request
+
+                {
+                    "_extra_data": {
+                        "validatior_errors": [
+                            {
+                                "path": ["path", "to", "error"],
+                                "message": "required: ['missing_key1', 'missing_key2']"
+                            }
+                        ],
+                        ... rest of extra data content
+                    },
+                    "id": 910648,
+                    "metadata": {
+                        "$schema": "https://labs.inspirehep.net/schemas/records/hep.json",
+                        ... record content
+                    }
+                }
+        """
+        workflow_data = _validate_workflow_payload(
+            ['metadata', '_extra_data', 'id'])
+        workflow_id = workflow_data['id']
+
+        try:
+            workflow = workflow_object_class.get(workflow_id)
+        except WorkflowsMissingObject:
+            raise CallbackWorkflowNotFoundError(workflow_id)
+
+        workflow.extra_data.pop('callback_url', None)
+        workflow.extra_data.pop('validation_errors', None)
+
+        workflow.data = workflow_data['metadata']
+        workflow.save()
+        db.session.commit()
+
+        if workflow.status != ObjectStatus.COMPLETED:
+            workflow.continue_workflow(delayed=True)
```

Comment:
This should change the status of the workflow to running too.

Reply:
change status from ``ERROR`` -> ``HALTED``?